docs/tutorials/github-guide.md

```diff
@@ -0,0 +1,186 @@
+---
+id: github-guide
+title: GitHub Guide 
+sidebar_label: GitHub Guide 
+---
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+[GitHub](https://docs.github.com/en/free-pro-team@latest/github) is a project management system in the cloud that allows developers to store and have code version control. This platform allows you to work collaboratively with other people around the world, track jobs, and manage projects.
+[Learn more about GitHub](https://www.howtogeek.com/180167/htg-explains-what-is-github-and-what-do-geeks-use-it-for/)  
+
+For EOS Costa Rica, it is of utmost importance that our collaborators familiarize themselves with this tool, which is why we offer below a basic guide to get started in the world of GitHub.
+
+
+* * *
+
+## **Tools**
+### [GitHub CLI](https://docs.github.com/es/free-pro-team@latest/github/getting-started-with-github/github-cli)
+
+GitHub [CLI](https://cli.github.com) is a command line tool for using GitHub functions on your own computer.
+
+<img alt="GitHub CLI Image" src={ useBaseUrl( 'img/LogosGitHub/GitHub-CLI.png' )} />
+
+### [GitHub Desktop](https://docs.github.com/es/free-pro-team@latest/github/getting-started-with-github/github-desktop)
+
+[GitHub Desktop](https://docs.github.com/en/desktop/installing-and-configuring-github-desktop/installing-github-desktop) is the desktop application that allows the user to interact with Github functionalities through the graphical interface, replacing the lines of code with a visual device.
+
+<img alt="GitHub Desktop Image" src={ useBaseUrl( 'img/LogosGitHub/GitHub-Desktop.png' )} />
+
+### [GitHub for mobile](https://docs.github.com/en/github/getting-started-with-github/github-for-mobile)
+
+
+[GitHub for mobile](https://play.google.com/store/apps/details?id=com.github.android&hl=es_419&gl=US) is available for Android and iOS, it gives you a way to get high-impact work on GitHub quickly, from anywhere. It also becomes a secure and stable way to access GitHub data through a trusted source client application.
+
+<img alt="GitHub Móvil Image" src={ useBaseUrl( 'img/LogosGitHub/GitHub-Movil.png' )} />
+
+### [Git Bash](https://desarrolloweb.com/articulos/entiende-instala-configura-git.html#:~:text=Git%20Bash%20es%20la%20línea,para%20usar%20Git%20en%20Windows.)
+
+[Git Bash](https://gitforwindows.org) is a Git application for Windows, where it allows working through command lines managing projects found on GitHub.
+
+<img alt="Git Bash Image" src={ useBaseUrl( 'img/LogosGitHub/GitHub-Bash.png' )} />
+
```

Comment:
There are also tools like Hub and git-extras that facilitate interacting with Github. You can leverage these tools to contribute to this repository.

---

Hay también herramientas como Hub y git-extras que facilitan la interacción con GitHub.

Please see : https://guias.eoscostarica.io/docs/pautas-para-codigo-abierto#herramientas-de-git-avanzadas

Reply:
Of course!!